resources/views/welcome.blade.php

```diff
@@ -56,21 +56,21 @@
                             <ul class="flex items-center mr-10">
                                 <li
                                     class="text-lg text-white transition duration-500 ease-in-out delay-150 font-heading mr-9 hover:-translate-y-1 hover:scale-110">
-                                    <a href="#">Nossa
+                                    <a href="#out-campaign">Nossa
                                         Campanha</a>
                                 </li>
                                 <li
                                     class="text-lg text-white transition duration-500 ease-in-out delay-150 font-heading mr-9 hover:-translate-y-1 hover:scale-110">
-                                    <a href="#">Como
+                                    <a href="#donation">Como
                                         Doar</a>
                                 </li>
                                 <li
                                     class="text-lg text-white transition duration-500 ease-in-out delay-150 font-heading mr-9 hover:-translate-y-1 hover:scale-110">
-                                    <a href="#">Doadores</a>
+                                    <a href="#donors">Doadores</a>
                                 </li>
                                 <li
                                     class="text-lg text-white transition duration-500 ease-in-out delay-150 font-heading hover:-translate-y-1 hover:scale-110">
-                                    <a href="#">Parceiros</a>
+                                    <a href="#partners">Parceiros</a>
```

Comment:
This part doesn't exist yet, can we delete?

Reply:
I'll implement it later. Thanks for remembering me about that.